### ISSUE TITLE
Seems not working on office365/Azure Information Protection?

### ISSUE THREAD

**Issue by lake2** [RUB-NDS/MS-RMS-Attacks#2]
Last year I received a protected document from my friend, and I tried to use the program to test it. It unprotected the file perfect. At that time my friend deployed Active directory Right Management on Server and protected that file. I'm not in the active Directory and I have the password to view and edit the document.
Today I received a document protected by office365/Azure Information Protection, and I tried to unprotected the file ( I'm not in the active Directory and I have the password to view and edit the document). It asked me to install Microsoft Online Services Sign-In Assistant for IT Professionals RTW  from http://aka.ms/sia, and I installed it and ran the program again. Then I encountered the error as the closed issue:
```
The application must provide a key handle to access the requested information. Contact your application support for further investigation.
The parameter is incorrect.
```
It there a way to fix it ? Thanks.

**Comment by Kaedo** [comment]
We always wanted, but never extended and tested the attack against the new AIP.

I will try to look at this next year.

**Comment by MinMorts** [comment]
I am also very interested in getting this working with AIP and office docs. I have figured out how to do the decryption of the standard pfiles, but not anything for office docs atm

**Comment by dnkrse** [comment]
The API authorization seems to be fixed - the concept here shouldn't work anymore. (luckely 👍)

**Comment by lake2** [comment]
> The API authorization seems to be fixed - the concept here shouldn't work anymore. (luckely 👍)

Does that means office365/Azure Information Protection is safe and cannot be unprotected?

**Comment by dnkrse** [comment]
Yes, it is mitigated:
https://techcommunity.microsoft.com/t5/Office-365-Encryption-AMA/Unauthorized-Removal-of-RMS-protection/m-p/222195

**Comment by Kaedo** [comment]
TL;DR
AIP, Azure RMS and AD RMS still have the same limitations as before, you can unprotect, modify and reprotect the content as you wish.
As of May 2020, it had no high priority at Microsoft to change this, if customer desire this feature this may change, until then everyone with read access can modify your documents, bulk decrypt all document he can read, and so on.
Find the new details in the corresponding [new repository](https://github.com/Kaedo/MS-AIP-Attacks).
